docs/examples/hello_world/README.md

```diff
@@ -0,0 +1,229 @@
+# Hello World application using GlusterFS Dynamic Provisioning
+
+At this point, we have a working Kubernetes cluster deployed, and a working Heketi Server.
+Next we will create a simple NGINX HelloWorld application utilizing Kubernetes Dynamic Provisioning and 
+Heketi.
+
+This example assumes some familiarity with Kubernetes and the [Kubernetes Persistent Storage](http://kubernetes.io/docs/user-guide/persistent-volumes/) model.
+
+
+### Verify our environment and gather some information to be used in later steps.
+
+Identify the Heketi REST URL and Server IP Address:
+
+```
+$ echo $HEKETI_CLI_SERVER
+http://10.42.0.0:8080
+```
+
+Identify the Gluster EndPoints that are needed to pass in as a parameter into the StorageClass (heketi-storage-endpoints):
+
+```
+kubectl get endpoints
+NAME                       ENDPOINTS                                            AGE
+heketi                     10.42.0.0:8080                                       22h
+heketi-storage-endpoints   192.168.10.100:1,192.168.10.101:1,192.168.10.102:1   22h
+kubernetes                 192.168.10.90:6443                                   23h
+```
+
+##### NOTE: EndPoints define the GlusterFS cluster, for version 1.4.X this is a required parameter for the StorageClass (Parameters are likely to change in later versions)
```

Comment:
Don't use sub-heading notation for notes, bold or italicize or something similar.